molgenis-js/src/main/resources/js/script-evaluator.js

```diff
@@ -1,452 +1,444 @@
 /**
```

Comment:
Do we have automated tests for all functions? Do these changes change the interface in the Java <-> Java or JavaScript <-> JavaScript scenarios? It is a bit hard to see what changed due to formatting changes.